### DISCUSSION TITLE
New icon for My Handy Product in Mein O2

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1954] [Proposals]
### Goal

Get a new icon for My Handy Product in Mein O2

### Context

Mein O2 have a product called My Handy where users can buy products such as devices, tablets, smartwatches. They use this icon on their website but we do not have this icon in our DS.
Mein O2 do not have the light versión.

### Impact

Align the website with the app.

### Discussion Questions

Do you need anything else?

### Do you have a visual proposal?

https://www.figma.com/design/SViPBhI1UNMlmvwwPh5Rvj/Mein-O2?node-id=34-2&t=4H5hBL5KIjcQufle-4

**Comment by AnaMontes11** [comment]
Hi @JoseTelefonica, we’ll include it. I’ll create an issue for this (#1964) and let you know as soon as it’s available.
